Docs/sphinx_documentation/source/Post_Processing.rst

```diff
@@ -20,7 +20,7 @@ row/column formatting, etc.
 
 
 fextract
-====================
+========
```

Comment:
Is there some utf8 character issue here?

Reply:
rst wants the length of the underline to match the length of the word(s)